src/app/components/wrapper-components/GameWrapper.ts

```diff
@@ -206,6 +234,19 @@ export abstract class GameWrapper<P extends Comparable> extends BaseWrapperCompo
         }
     }
 
+    /**
+     * To use when a previous move exist when the user provoked a move by clicking
+     * Since he clicked, the hideLastMove is no longer necessary
```

Comment:
they!
the call to hideLastMove ?  et pas clair l'utilité de cette phrase, pourquoi on me parle de hideLastMove dans showNextMove, alors qu'il appelle clairement pas hideLastMove ?